src/data-types/smallint.ts

```diff
@@ -1,6 +1,8 @@
+import { DataType } from '../data-type';
+
 const IntN = require('./intn');
 
-module.exports = {
+const SmallInt: & DataType = {
```

Comment:
I think there's still a leftover `&` here. 😬 

```suggestion
const SmallInt: DataType = {
```